doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,42 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the networking core image
+******************************************************
+
+In addition to the recovery of the application core image, also the networking core image can be recovered.
+You can use this feature if you build nRF53 MCUboot with one of the following options:
+
+* :kconfig:option:`CONFIG_NRF53_MULTI_IMAGE_UPDATE` - Simultaneous multi-image DFU.
+* :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` - Serial recovery for the network core in the one image pair mode.
```

Comment:
What is the "one image pair mode" and where can I find more info about it? I see no results when looking for it in MCUboot docs and NCS docs...

Reply:
one image pair mode is the mode when UPDATEABLE_IMAGE_NUMBER == 1, so most popular dual-bank mode.

Reply:
```suggestion
* :kconfig:option:`CONFIG_NRF53_RECOVERY_NETWORK_CORE` - Serial recovery for the network core in the one image pair mode, where :kconfig:option:`CONFIG_UPDATEABLE_IMAGE_NUMBER` is set to ``== 1``.
  
```